diff --git a/deps/boost/libs/stacktrace/CMakeLists.txt b/deps/boost/libs/stacktrace/CMakeLists.txt
--- a/deps/boost/libs/stacktrace/CMakeLists.txt
+++ b/deps/boost/libs/stacktrace/CMakeLists.txt
@@ -98,6 +98,9 @@ stacktrace_add_library(backtrace ${BOOST_STACKTRACE_ENABLE_BACKTRACE} "backtrace;${CMAKE_DL_LIBS}" "")
 stacktrace_add_library(addr2line ${BOOST_STACKTRACE_ENABLE_ADDR2LINE} "${CMAKE_DL_LIBS}" "")
 stacktrace_add_library(basic ${BOOST_STACKTRACE_ENABLE_BASIC} "${CMAKE_DL_LIBS}" "")
+if(MINGW)
+  target_link_libraries(boost_stacktrace_basic PRIVATE ntdll)
+endif()
 stacktrace_add_library(windbg ${BOOST_STACKTRACE_ENABLE_WINDBG} "dbgeng;ole32" "_GNU_SOURCE=1")
