diff --git a/deps/abseil-cpp/absl/time/internal/cctz/src/time_zone_lookup.cc b/deps/abseil-cpp/absl/time/internal/cctz/src/time_zone_lookup.cc
--- a/deps/abseil-cpp/absl/time/internal/cctz/src/time_zone_lookup.cc
+++ b/deps/abseil-cpp/absl/time/internal/cctz/src/time_zone_lookup.cc
@@ -40,6 +40,7 @@
 // Include only when the SDK is for Windows 10 (and later), and the binary is
 // targeted for Windows XP and later.
 #if defined(_WIN32_WINNT_WIN10) && (_WIN32_WINNT >= _WIN32_WINNT_WINXP)
 #include <roapi.h>
+#if defined(__MINGW32__)
+extern "C" {
+HRESULT WINAPI WindowsCreateStringReference(PCWSTR, UINT32, HSTRING_HEADER*, HSTRING*);
+HRESULT WINAPI WindowsDeleteString(HSTRING);
+PCWSTR WINAPI WindowsGetStringRawBuffer(HSTRING, UINT32*);
+}
+#else
+#include <winstring.h>
+#endif
 #include <tchar.h>
