diff --git a/deps/tiles/include/tiles/util.h b/deps/tiles/include/tiles/util.h
--- a/deps/tiles/include/tiles/util.h
+++ b/deps/tiles/include/tiles/util.h
@@ -23,7 +23,7 @@ inline void t_log(fmt::format_string<Args...> fmt_str, Args&&... args) {
   auto const now = clock::to_time_t(clock::now());
   struct tm tmp{};
-#if _MSC_VER >= 1400
+#if defined(_MSC_VER) || defined(_WIN32)
   gmtime_s(&tmp, &now);
 #else
   gmtime_r(&now, &tmp);
