diff --git a/deps/res/create_resource.cmake b/deps/res/create_resource.cmake
--- a/deps/res/create_resource.cmake
+++ b/deps/res/create_resource.cmake
@@ -263,6 +263,11 @@ function(create_resource root input_paths lib)
   else()
     set(resource-linker ld)
   endif()
+  if (WIN32)
+    set(resource-linker-flags)
+  else()
+    set(resource-linker-flags -z noexecstack)
+  endif()
   if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "i686")
     set(flags "-melf_i386")
   endif()
@@ -277,7 +282,7 @@ reinterpret_cast<void const*>(&_binary_${mangled-path}_start)},\n")
     string(APPEND emplace-statements "    m.emplace(\"${rel-path}\", ${id});\r\n")
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${lib}/obj)
     add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${lib}/obj/res_${id}.o
-      COMMAND ${resource-linker} ${flags} -z noexecstack -r -b binary -o ${CMAKE_CURRENT_BINARY_DIR}/${lib}/obj/res_${id}.o ${rel-path}
+      COMMAND ${resource-linker} ${flags} ${resource-linker-flags} -r -b binary -o ${CMAKE_CURRENT_BINARY_DIR}/${lib}/obj/res_${id}.o ${rel-path}
       DEPENDS ${p}
       WORKING_DIRECTORY ${root}
       COMMENT "Generating resource ${rel-path} (${CMAKE_CURRENT_BINARY_DIR}/${lib}/obj/res_${id}.o)"
@@
diff --git a/deps/conf/src/date_time.cc b/deps/conf/src/date_time.cc
--- a/deps/conf/src/date_time.cc
+++ b/deps/conf/src/date_time.cc
@@ -8,7 +8,7 @@
 #include "boost/algorithm/string.hpp"
 #include "boost/date_time/posix_time/posix_time_types.hpp"
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(_WIN32)
 #define gmt(a, b) gmtime_s(b, a)
 #else
 #define gmt(a, b) gmtime_r(a, b)
