diff --git a/deps/tg/CMakeLists.txt b/deps/tg/CMakeLists.txt
--- a/deps/tg/CMakeLists.txt
+++ b/deps/tg/CMakeLists.txt
@@ -2,9 +2,12 @@ cmake_minimum_required(VERSION 3.20)
 project(tg)
 
 add_library(tg tg.c)
+if (MINGW)
+    set_source_files_properties(tg.c PROPERTIES COMPILE_OPTIONS "-std=gnu11")
+endif()
 target_include_directories(tg PUBLIC .)
 
 #if (MSVC)
 #    set_source_files_properties(tg PROPERTIES LANGUAGE CXX)
 #    target_compile_features(tg PRIVATE cxx_std_23)
-#endif()
\ No newline at end of file
+#endif()
