diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -190,6 +190,10 @@ target_link_libraries(motis
     address_formatting_res-res
 )
 
+if(MINGW)
+    add_dependencies(motis tiles_server_res)
+endif()
+
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
     target_link_options(motis PRIVATE -Wl,-no_deduplicate)
 endif()
@@ -216,7 +220,7 @@ target_compile_options(motis-test PRIVATE ${motis-compile-options})
 set_property(
     TARGET motis tiles tiles-import-library
     APPEND PROPERTY COMPILE_DEFINITIONS TILES_GLOBAL_PROGRESS_TRACKER=1)
-file (CREATE_LINK ${CMAKE_SOURCE_DIR}/deps/tiles/profile ${CMAKE_BINARY_DIR}/tiles-profiles SYMBOLIC)
+file (MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/tiles-profiles)
 
 # --- MIMALLOC ---
 if (MOTIS_MIMALLOC)
@@ -266,7 +270,7 @@ add_custom_target(motis-web-ui
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/ui"
         VERBATIM
 )
-file (CREATE_LINK ${CMAKE_SOURCE_DIR}/ui/build ${CMAKE_BINARY_DIR}/ui SYMBOLIC)
+file (MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/ui)
 
 foreach(t adr osr nigiri gtfsrt
         geo tiles tiles-import-library
